package.json

```diff
@@ -16,7 +16,8 @@
   "scripts": {
     "start": "NODE_ENV=production node ./src/server/server.babel.js",
     "dev": "NODE_ENV=development node ./src/server/server.babel.js",
-    "build": "rimraf build && concurrently \"npm run build:client\" \"npm run build:server\"",
+    "build:themes": "babel-node ./webpack/utils/buildThemeJSON.js",
```

Comment:
Did you prefer building the theme here over using `WebpackShellPlugin`? I'm cool with this of course, just curious as to why...